Jenkinsfile

```diff
@@ -12,6 +12,11 @@ pipeline {
           branch 'pr-jenkins' // for testing
         }
       }
+      not {
+        changeSet {
+          files('docs/**') // Don't run for changes under 'docs'
+        }
+      }
```

Comment:
Actually this is giving me an error. 

@mrpollo This is not quite right anyway. 
- We don't want everything to run again if we make a change to the /docs folder - which we do by pushing these changes.
- Ideally we don't want anything in this folder to run for changes that are **only** to /docs/ folder - so it is OK to run if we have a changeset where at least one extra file outside of docs is running.

Please take this over :-)